packages/devextreme/js/__internal/ui/chat/chat_message_box.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import $ from '@js/core/renderer';
+import Button from '@js/ui/button';
+
+import TextArea from '../m_text_area';
+import Widget from '../widget';
+
+const CHAT_MESSAGE_BOX_CLASS = 'dx-chat-message-box';
+const CHAT_MESSAGE_BOX_TEXTAREA_CLASS = 'dx-chat-message-box-text-area';
+const CHAT_MESSAGE_BOX_BUTTON_CLASS = 'dx-chat-message-box-button';
+
+class MessageBox extends Widget<any> {
+  // TODO chech ts
+  _textArea?: any;
+
+  _button?: Button;
+
+  _initMarkup(): void {
+    $(this.element()).addClass(CHAT_MESSAGE_BOX_CLASS);
+
+    super._initMarkup();
+
+    this._renderTextArea();
```

Comment:
[SSR]
Kindly ensure that your markup tests are run in server side mode. Because now I see that you render everything inside of `_initMarkup` and nothing inside of `_renderContent`. Mostly it's okay but need to be checked

Reply:
I've added SSR testing. It was successful